docs/api/api.yml

```diff
@@ -0,0 +1,83 @@
+openapi: 3.0.0
+info:
+  title: Download Service API
+  version: 1.1.2
+  description: |
+    This document specifies the Download Service API. It provides a direct interface to download files from the archive.
+    This API implements the operations `GET` on the paths `/` and `/ark:/:NAAN/:uuid/*`.
+    The NAAN represents the configured 'name assigning authority number' which for DANS is 73189.
+    It is not neccessary for this service to supply the bag store the bag is stored in.
+
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0
+
+servers:
+  - url: 'http://localhost:20160'
+  - url: 'http://test.dans.knaw.nl:20160'
+
+paths:
+  /:
+    get:
+      summary: Returns a description of the service.
+      description:
+        This is the starting place for navigating the Download Service. It returns a description of the service.
+      responses:
+        200:
+          description: Description of the service
+          content:
+            text/plain:
+              schema:
+                type: string
+                description: A plain text, human readable description of the service
+                example: Download Service is running.
+        500:
```

Comment:
when do you get a `500` with this endpoint?

Reply:
I would imagine this could occur if one of the services this depends on returns an error or a CompositeError occurs in one of the methods invoked by this. Theoretically it SHOULDN'T happen. 

to be honest, choosing to include or not include this is a recurring question I have had, to which there has been no clear answer yet. The question, more specifically is: Do you include general status codes, and to which extent do you include them. There is documentation of other services which already omitted the 408, because it shouldn't occur on a regular basis and feels like a 'general response code', which is why it also wasn't originally included in this documentation. 

In short, the convention of when to include or not to include general status codes is unclear right now.

Reply:
I'd say a nice convention is: "_if it cannot occur, don't put it in_"... Especially with these documentations, where you're basically reverse engineering the documentation from the code, I think you should read the code and see which status can/cannot be returned from a route.

https://github.com/DANS-KNAW/easy-download/blob/a91e45559027d68e9ee8603403daf3fb302dd91a/src/main/scala/nl.knaw.dans.easy.download/EasyDownloadServlet.scala#L40-L43

Given this to be the implementation for `GET /`, I can only conclude the only suitable response code to be `200`.